models/patientsymptom.model.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const patientsymptomSchema = new Schema({
+
+    patientFullName: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    date: {
+        type:Date,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom1: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom2: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom3: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom4: {
+        type: String,
+      //  required: true,
+        
+       // unique: true
```

Comment:
Symptom 1- 4 are Yes or No questions, so having booleans instead of strings might work better

Reply:
Yes you are I will change that